eng/scripts/CreateVSHive.ps1

```diff
@@ -18,7 +18,7 @@ Set-StrictMode -Version 1
 $success=$false
 for($i=0; $i -le 3; $i++)
 {
-  & $devenvExePath /rootsuffix $rootSuffix /updateConfiguration
+  & $devenvExePath /rootsuffix $rootSuffix /updateConfiguration /log
```

Comment:
Yeah @davidwengier is right - this is only going to add the log param for the call to `/updateConfiguration`, not the actual integration test instance.  

It looks like though you might be able to set an environment variable to configure the level instead?
See https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=/src/product/RemoteLanguage/Impl/Platform/VSWindows/HostServices/TraceSourceHelper.cs&version=GBdevelop&line=74&lineEnd=75&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
Thanks! I updated it to set the environment variable - hopefully it looks right?

Reply:
Somehow I'm unable to push the branch to the internal AzDo repo to verify whether this works 😅